stampit.js

```diff
@@ -189,16 +191,21 @@ isStamp = function isStamp(obj) {
  * @return {Function}             A composable stampit factory
  *                                (aka stamp).
  */
-convertConstructor = function convertConstructor(Constructor) {
-  return stampit().methods(Constructor.prototype).enclose(Constructor);
-};
+function convertConstructor(Constructor) {
+  var stamp = stampit();
+  mixer.mixInChainFunctions(stamp.fixed.methods, Constructor.prototype);
+  mixer.mixIn(stamp.fixed.state, Constructor);
+  stamp.fixed.state = mixer.mergeChainObjects(stamp.fixed.state, Constructor.prototype);
```

Comment:
Isn't it true that this will still override any method copied to `stamp.fixed.methods`? So this makes line 196 useless...